### PR TITLE
SPUI-1261 > Change allow camera ui

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart [hunk 1]
```diff
@@ -68,15 +69,27 @@ class _AllowCameraState extends State<AllowCamera> with WidgetsBindingObserver {
...
     return SPageFrameWithPadding(
-      header: SMegaHeader(
-        titleAlign: TextAlign.left,
-        title: state.permissionDenied
-            ? 'Give permission to\nallow to use camera'
-            : 'Allow camera access',
+      header: deviceSize.when(
+        small: () {
+          return SSmallHeader(
+            title: state.permissionDenied
```
Comment (eli1stark) [inline lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart:80]: refactor title to function and name it `headerTitle()`

File: lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart [hunk 2]
```diff
@@ -68,15 +69,27 @@ class _AllowCameraState extends State<AllowCamera> with WidgetsBindingObserver {
     final colors = useProvider(sColorPod);
     final state = useProvider(cameraPermissionNotipod);
     final notifier = useProvider(cameraPermissionNotipod.notifier);
+    final deviceSize = useProvider(deviceSizePod);
 
     analytics(() => sAnalytics.kycAllowCameraView());
```
Comment (eli1stark) [inline lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart:75]: add `final size  = MediaQuery.of(context).size;`

File: lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart [hunk 3]
```diff
@@ -103,6 +116,7 @@ class _AllowCameraState extends State<AllowCamera> with WidgetsBindingObserver {
                 const Spacer(),
                 Image.asset(
                   allowCameraAsset,
+                  height: MediaQuery.of(context).size.width * 0.6,
```
Comment (eli1stark) [inline lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart:119]: add `size.width * 0.6` instead

File: lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart [hunk 4]
```diff
@@ -68,15 +69,27 @@ class _AllowCameraState extends State<AllowCamera> with WidgetsBindingObserver {
...
+          return SSmallHeader(
+            title: state.permissionDenied
+                ? 'Give permission to\nallow to use camera'
+                : 'Allow camera access',
+          );
+        },
+        medium: () {
+          return SMegaHeader(
+            titleAlign: TextAlign.left,
+            title: state.permissionDenied
```
Comment (eli1stark) [inline lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart:88]: reuse private function

File: lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart (hunk 2 above)
Comment (eli1stark) [reply lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart:75]: you can declare function `headerTitle()` after `size` declaration, don't forget space between them

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart (hunk 1 above)
Comment (DenisMartych) [reply lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart:80]: done

File: lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart (hunk 2 above)
Comment (DenisMartych) [reply lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart:75]: done

File: lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart (hunk 3 above)
Comment (DenisMartych) [reply lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart:119]: done

File: lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart [hunk 5]
```diff
@@ -140,4 +152,10 @@ class _AllowCameraState extends State<AllowCamera> with WidgetsBindingObserver {
       ),
     );
   }
+
+  String _headerTitle(bool status) {
+    return status
```
Comment (eli1stark) [inline lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart:157]: use if statement instead

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart (hunk 5 above)
Comment (DenisMartych) [reply lib/app/shared/features/kyc/view/components/allow_camera/allow_camera.dart]: done

**Pull request merged**